frontend/src/libs/components/input/input.tsx

```diff
@@ -44,12 +46,24 @@ const Input = <T extends FieldValues>({
   onBlur,
   rows,
   isInline,
+  tooltip,
+  isTooltipShowedOnlyOnError = false,
 }: Properties<T>): JSX.Element => {
   const { field } = useFormController({ name, control });
 
   const error = errors[name]?.message;
   const hasError = Boolean(error);
 
+  const shouldShowTooltip = isTooltipShowedOnlyOnError ? hasError : true;
```

Comment:
Won't `!isTooltipShowedOnlyOnError || hasError` work the same?

Reply:
Yeah, it will. For some reason I thought that due to linter I should always use ?? in this type of conditions